dynparquet/example.go

```diff
@@ -50,6 +53,36 @@ func (s Samples) ToBuffer(schema *Schema) (*Buffer, error) {
 	return pb, nil
 }
 
+func (s Samples) ToRecord(schema *arrow.Schema) (arrow.Record, error) {
+	bld := array.NewRecordBuilder(memory.NewGoAllocator(), schema)
+	defer bld.Release()
+
+	numLabels := len(schema.Fields()) - 4
```

Comment:
It would be nice to get a comment on why the `- 4` is needed. Is it because of the stack trace, timestamp, and value addition at the end?

Reply:
![](https://media.giphy.com/media/12NUbkX6p4xOO4/giphy.gif)